algorithms/src/Kokkos_StdAlgorithms.hpp

```diff
@@ -24,91 +24,91 @@
 /// \file Kokkos_StdAlgorithms.hpp
 /// \brief Kokkos counterparts for Standard C++ Library algorithms
 
-#include "std_algorithms/impl/Kokkos_Constraints.hpp"
-#include "std_algorithms/impl/Kokkos_RandomAccessIterator.hpp"
-#include "std_algorithms/Kokkos_BeginEnd.hpp"
+#include "std_algorithms/impl/Kokkos_Constraints.hpp"  // IWYU pragma: export
+#include "std_algorithms/impl/Kokkos_RandomAccessIterator.hpp"  // IWYU pragma: export
```

Comment:
These have `impl` in their name and shouldn't be included explicitly in downstream code.